### PR TITLE
App Submission: Blocky

### PR DESCRIPTION
**Pull request by awholegnuworld**
This isn't working in Multipass or my Umbrel, but it could be because I have Pi-Hole installed on my main Umbrel. It complains about port 53 being taken even on a clean Multipass install.

### CODE REVIEW

File: blocky/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  blocky:
```
Comment (IMPranshu) [inline blocky/docker-compose.yml:3]:
```suggestion
  app_proxy:
    environment:
      APP_HOST: blocky_web_1
      APP_PORT: 4000
  web:
```

File: blocky/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    image: spx01/blocky
+    container_name: blocky
+    restart: unless-stopped
+    # Optional the instance hostname for logging purpose
+    hostname: blocky-hostname
+    ports:
```
Comment (IMPranshu) [inline blocky/docker-compose.yml:9]:
```suggestion
```

**Review by IMPranshu — CHANGES_REQUESTED** [review]

**Comment by IMPranshu** [comment]
I have left some comments do try them and see if the error occurs again.

**Comment by awholegnuworld** [comment]
Now would be a great time to say that I'm awful at Docker. 😄 I don't know if there is an arm64 image for the front end of this app, but the repository says there is. Ideas?

**Pull request closed without merging**